doc/source/news.rst

```diff
@@ -7,6 +7,79 @@
+  ``grndb recover --force-truncate`` option that it can be truncated even if
+  locks are left on the table.
+
+* [Windows] Use ``CONDITION_VARIABLE`` instead of Groonga's original implementation.
+
+* [:doc:`/reference/commands/logical_range_filter`] Added ``sort_keys`` option.
+
+* Added a new function ``time_format()``.
+  You can specify time format against a column of Time type.
+  You can specify with use format of ``strftime`` .
+
+* [:doc:`/reference/tokenizers`] Added the tokenizer that to be able to set options.
+  You can dynamically change tokenizer's behavior.
+  Specific options is as below.
+  * ``n`` : You can specify "N" of Ngram. For example, Ngram is trigram
+    when you can set "3" to "N".
+  * ``loose_symbol`` : The keywords including symbols are hit, even the
+    keywords not including symbols are hit also.
+  * ``remove_blank`` : Ignore blank.
+  * ``loose_blank`` : The keywords including blank are hit, even the
+    keywords not including blank are hit also.
+
+* [:doc:`/reference/normalizers`] Support Unicode 10.0
+
+* [:doc:`/reference/normalizers`] Added the normalizers that to be able to set options.
+  You can dynamically change normalizer's behavior.
+  Specific options is as below.
+  * ``unify_kana`` : Full-width half Hiragana and Full-width half Katakana are
+    regarded as the same character.
+  * ``unify_kana_case`` : Uppercase and lowercase letters of Full-width half Hiragana
+    and Full-width half Katakana are regarded as the same character.
+  * ``unify_kana_voiced_sound_mark`` : Voiced sound mark and semi voiced sound mark of
+    Full-width half Hiragana and Full-width half Katakana are regarded as the same character.
+  * ``unify_hyphen`` : The characters like hyphen are regarded as the hyphen.
+  * ``unify_prolonged_sound_mark`` : The characters like prolonged sound mark are regarded
+    as the prolonged sound mark.
+  * ``unify_hyphen_and_prolonged_sound_mark`` : The characters like hyphen and prolonged sound
+    mark are regarded as the hyphen.
+
+* [:doc:`/reference/commands/dump`] Support output of tokenizer's options and normalizer's options.
+  If you import dump that makes by Groonga 8.0.2 or later against a database that makes by Groonga
+  earlier than 8.0.2, occurs error.
+  Because database that makes by Groonga earlier than 8.0.2 is not supported tokenizer's options and
+  normalizer's options.
+
+* [:doc:`/reference/commands/schema`] Support output of tokenizer's options and normalizer's options.
+  If you import schema that makes by Groonga 8.0.2 or later against a database that makes by Groonga
+  earlier than 8.0.2, occurs error.
+  Because database that makes by Groonga earlier than 8.0.2 is not supported tokenizer's options and
+  normalizer's options.
+
+* [httpd] Updated bundled nginx to 1.13.12.
+
+Fixes
+^^^^^
+
+* Fixed a bug that unexpected record is matched with space only query.
+  [groonga-dev,04609][Reported by satouyuzh]
+
```

Comment:
It seems that https://github.com/groonga/groonga/commit/e8203c5fe805fcb4d2aebab3a6fdc5b628d27178 entry is missing from news.

Reply:
Ubuntu 18.04 support is also missing?

Reply:
It seems that msgpack-c < 1 was dropped. am i wrong?

Reply:
It's not dropped. If Groonga isn't buildable with msgpack-c < 1, it's a bug.

Reply:
Thanks, I got it.